docs/encyclopedia/workflows.mdx

```diff
@@ -449,6 +449,17 @@ These limits are set with the following [dynamic configuration keys](https://git
 - `NumPendingSignalsLimit`
 - `NumPendingCancelRequestsLimit`
 
+#### Workflow Execution Nexus Operation Limits {#workflow-execution-nexus-operation-limits}
+
+There is also a limit to the maximum number of Nexus Operations in a Workflow before Continue-As-New is required.
+Each in-progress Nexus Operation generates a metadata entry in the Workflow Execution's mutable state. Too many entries in a single Workflow Execution's mutable state causes unstable persistence.
+To protect the system, Temporal enforces a maximum number of incomplete (and complete in public preview) Nexus Operation requests per Workflow Execution (by default, 30 Nexus Operations).
+Once the limit is reached for a type of operation, if the Workflow Execution attempts to start another Nexus operation (by producing a ScheduleNexusOperation), it will be unable to (the Workflow Task Execution will fail and get retried).
```

Comment:
```suggestion
Once the limit is reached for a type of operation, if the Workflow Execution attempts to start another Nexus operation (by producing a ScheduleNexusOperation), it will be unable to do so (the Workflow Task Execution will fail and get retried).
```

Reply:
nice!